test/suite/contextValue.test.ts

```diff
@@ -11,75 +11,90 @@ import { ContainerNode, FileNode, FolderNode, INodeData, NodeKind, PackageNode,
 suite("Context Value Tests", () => {
 
     test("test workspace node", async function() {
-        assert.equal((await workspace.getTreeItem()).contextValue, "java:workspaceFolder+uri");
+        assert.ok(/java:workspaceFolder(?=.*?\b\+uri\b)/.test((await workspace.getTreeItem()).contextValue || ""));
     });
 
     test("test Maven project node", async function() {
-        assert.equal((await mavenProject.getTreeItem()).contextValue, "java:project+java+maven+uri");
+        assert.ok(/java:project(?=.*?\b\+java\b)(?=.*?\b\+maven\b)(?=.*?\b\+uri\b)/.test((await mavenProject.getTreeItem()).contextValue || ""));
     });
 
     test("test Gradle project node", async function() {
-        assert.equal((await gradleProject.getTreeItem()).contextValue, "java:project+java+gradle+uri");
+        assert.ok(/java:project(?=.*?\b\+java\b)(?=.*?\b\+gradle\b)(?=.*?\b\+uri\b)/.test((await gradleProject.getTreeItem()).contextValue || ""));
     });
 
     test("test JRE container node", async function() {
-        assert.equal((await jreContainer.getTreeItem()).contextValue, "java:container+jre+uri");
+        assert.ok(/java:container(?=.*?\b\+jre\b)(?=.*?\b\+uri\b)/.test((await jreContainer.getTreeItem()).contextValue || ""));
     });
 
     test("test Maven container node", async function() {
-        assert.equal((await mavenContainer.getTreeItem()).contextValue, "java:container+maven+uri");
+        assert.ok(/java:container(?=.*?\b\+maven\b)(?=.*?\b\+uri\b)/.test((await mavenContainer.getTreeItem()).contextValue || ""));
     });
 
     test("test Gradle container node", async function() {
-        assert.equal((await gradleContainer.getTreeItem()).contextValue, "java:container+gradle+uri");
+        assert.ok(/java:container(?=.*?\b\+gradle\b)(?=.*?\b\+uri\b)/.test((await gradleContainer.getTreeItem()).contextValue || ""));
```

Comment:
suggest to assign human-readable names to these regular expressions...

Reply:
These are all the regexp that can be written in the package.json to register the commands to the node.

I add some comments at header to explain the purpose of this suite.